### ISSUE TITLE
Choosing colors by entering color codes such as hex codes or rgba

### ISSUE THREAD

**Issue by fuji-184** [termux/termux-styling#230]
**Feature description**
How about providing an input for color codes like hex codes or rgba so that the terminal color follows the input color code

**Reference implementation**
Unfortunately, I am a beginner, so I don't know how to implement it

**Comment by Hymnu** [comment]
> You can already do this by picking some theme and then editing `~/.termux/colors.properties` file.

Certainly. However, I can't seem to get any information that explains the meanings of the options in the "colors.properties" file. Additionally, the number of options varies across different themes. For instance, the "default" theme doesn't display any options in the "colors.properties" file. 😔
I attempted to display them in an HTML format, but ultimately identify only a few of them.😭😭😭
